src/gui/tutorial.cpp

```diff
@@ -31,6 +31,13 @@
 #include <dirent.h>
 #endif
 
+#define CLICK_TO_COPY(t) ImGui::TextColored(uiColors[GUI_COLOR_ACCENT_PRIMARY],t);\
+if (ImGui::IsItemHovered()) {\
+  ImGui::SetTooltip((strcmp(ImGui::GetClipboardText(),t)==0)?"copied!":"click to copy");\
```

Comment:
May be slow.

By the way, by using SDL_OpenURL (?) or something we could make this open the user's browser.

Reply:
i shall replace it with a proper "copied!" popup